doc/barman.1.d/50-show-backup.md

```diff
@@ -36,4 +47,39 @@ Backup 20150828T130001:
     Retention Policy     : not enforced
     Previous Backup      : 20150821T130001
     Next Backup          : - (this is the latest base backup)
+    Root Backup          : 20240814T015504
+    Parent Backup        : 20240814T016504
+    Backup chain size    : 3
+    Children Backup(s)   : 20240814T018515
 ```
+
```

Comment:
Please, this is looking promising Please, take another look for feedback/suggestions! @gcalacoci @barthisrael @bleidens @gustabowill

Reply:
tbh everything LGTM